packages/typescript/README.md

```diff
@@ -79,6 +79,15 @@ Default: `null`
 
 A [minimatch pattern](https://github.com/isaacs/minimatch), or array of patterns, which specifies the files in the build the plugin should operate on. By default all `.ts` and `.tsx` files are targeted.
 
+### `filterRoot`
+
+Type: `String` | `Boolean`<br>
+Default: `true`
```

Comment:
I think "`rootDir` in `tsconfig.json`" or "`process.cwd`" makes more sense as the default value written in the docs.

Reply:
Fixed